yt/yt/server/master/security_server/user.h

```diff
@@ -213,6 +213,8 @@ class TUser
     DEFINE_BYREF_RO_PROPERTY(std::optional<TString>, HashedPassword);
     //! Salt used for password encryption.
     DEFINE_BYREF_RO_PROPERTY(std::optional<TString>, PasswordSalt);
+    //! A flag that indicates that password is temporary and should be changed.
+    DEFINE_BYVAL_RW_PROPERTY(bool, PasswordIsTemporary);
```

Comment:
Either this attribute should be system and be a part of a master state (in which case it should be persisted), or it should be user attribute (in which case it should not be stored as a separate field). I think the second option is better here, as this attribute does not have any custom behavior on get/set/remove.

cc @gritukan

Reply:
Discussed with @gritukan, made it a user attribute